modules/storage-persistent-storage-pv.adoc

```diff
@@ -163,9 +168,11 @@ endif::[]
 [.small]
 --
 1. ReadWriteOnce (RWO) volumes cannot be mounted on multiple nodes. If a node fails, the system does not allow the attached RWO volume to be mounted on a new node because it is already assigned to the failed node. If you encounter a multi-attach error message as a result, force delete the pod on a shutdown or crashed node to avoid data loss in critical workloads, such as when dynamic persistent volumes are attached.
+ifndef::microshift[]
 2. Use a recreate deployment strategy for pods that rely on AWS EBS.
 3. If the underlying vSphere environment supports the vSAN file service, then the vSphere Container Storage Interface (CSI) Driver Operator installed by
 {product-title} supports provisioning of ReadWriteMany (RWX) volumes. If you do not have vSAN file service configured, and you request RWX, the volume fails to get created and an error is logged. For more information, see "Using Container Storage Interface" -> "VMware vSphere CSI Driver Operator".
+endif::microshift[]
```

Comment:
I still see footnote 3 in the table above for microshift. So should this endif be moved above 3 so that it shows?

Reply:
Table needs entirely removed. Fixed.